File: js/9. Palindrome Number.js
/**
 * @param {number} x
 * @return {boolean}
 */
const isPalindrome = function (x) {
    let str = String(x);
    const strLen = str.length;
    for (let i = 0; i < strLen; i++) {
        if (str[i] !== str[strLen - 1 - i]) {
            return false;
        }
    }
    return true;
};

console.log("test")

File: go/9. Palindrome Number.go
package main

import (
	"fmt"
	"strconv"
)

func isPalindrome(x int) bool {
	str := strconv.Itoa(x)
	strLen := len(str)
	for i := 0; i < strLen/2; i++ {
		if str[i] != str[strLen-i-1] {
			return false
		}
	}
	return true
}
func main() {
	fmt.Println(isPalindrome(1212123))
}
